index.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const Hapi = require('hapi');
+var models = require('./models');
+
+const init = async () => {
+
+    const server = Hapi.server({
+        port: 3000,
+        host: 'localhost'
+    });
+
+    server.route(require('./lib/routes'));
+
+    models.sequelize.sync().then(function() {
+      server.start(function() {
+        console.log('Running on 3000');
+      });
+    });
+};
```

Comment:
This is the block that you said we're probably end up doing it another way right?